### ISSUE TITLE
CombBLAS support file s_c2cpp_GetHWPM.cpp is missing

### ISSUE THREAD

**Issue by drew-parsons** [xiaoyeli/superlu_dist#110]
SRC/CMakeLists.txt includes s_c2cpp_GetHWPM.cpp if SuperLU-Dist is built against CombBlas (setting HAVE_COMBBLAS).

And SRC/psgssvx.c and other psg files do use s_c2cpp_GetHWPM. s_c2cpp_GetHWPM is referenced also in  SRC/superlu_sdefs.h

But s_c2cpp_GetHWPM.cpp does not exist. Both d_c2cpp_GetHWPM.cpp and z_c2cpp_GetHWPM.cpp are in the SRC dir, but s_c2cpp_GetHWPM.cpp  is not.

This causes a cmake build of SuperLU-Dist 8.0.0 (on Debian linux) to fail:
```
	cd obj-x86_64-linux-gnu && cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_INSTALL_RUNSTATEDIR=/run -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DBUILD_SHARED_LIBS=ON -DCMAKE_SKIP_RPATH=ON -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpic\+\+ -DXSDK_ENABLE_Fortran=ON -DCMAKE_FORTRAN_COMPILER=mpifort -DMPIEXEC_PREFLAGS=--allow-run-as-root -DCMAKE_INSTALL_INCLUDEDIR=include/superlu-dist -Denable_complex16=ON -DTPL_ENABLE_INTERNAL_BLASLIB=OFF -DTPL_BLAS_LIBRARIES=/usr/lib/x86_64-linux-gnu/libblas.so -DTPL_ENABLE_LAPACKLIB=ON -DTPL_LAPACK_LIBRARIES=/usr/lib/x86_64-linux-gnu/liblapack.so -DTPL_ENABLE_PARMETISLIB=ON "-DTPL_PARMETIS_LIBRARIES=-lparmetis -lmetis" -DTPL_PARMETIS_INCLUDE_DIRS=/usr/include/parmetis -DTPL_ENABLE_COMBBLASLIB=ON "-DTPL_COMBBLAS_LIBRARIES=-lCombBLAS -lGraphGenlib -lUsortlib" -DTPL_COMBBLAS_INCLUDE_DIRS=/usr/include/CombBLAS/ ..
-- The C compiler identification is GNU 11.3.0
...
-- superlu_dist_fortran will be built as a dynamic library.
-- Enabled support for COMBBLAS
-- Detecting Fortran/C Interface
-- Detecting Fortran/C Interface - Found GLOBAL and MODULE mangling
-- Verifying Fortran/CXX Compiler Compatibility
-- Verifying Fortran/CXX Compiler Compatibility - Success
-- Configuring done
CMake Error at SRC/CMakeLists.txt:277 (add_library):
  Cannot find source file:

    s_c2cpp_GetHWPM.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc


CMake Error at SRC/CMakeLists.txt:277 (add_library):
  No SOURCES given to target: superlu_dist


CMake Error at SRC/CMakeLists.txt:279 (add_library):
  No SOURCES given to target: superlu_dist-static


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by drew-parsons** [comment]
The reference to s_c2cpp_GetHWPM.cpp in SRC/CMakeLists.txt was added in commit
https://github.com/xiaoyeli/superlu_dist/commit/daab26087fa02662092cadc7fce8595b7c49586c#diff-af3c34819daa95011f4cc8590b1a6a03fda84959e9f392c577216614b736169c

**Comment by xiaoyeli** [comment]
This is fixed in the new release 8.1.0

**Comment by drew-parsons** [comment]
Confirmed.
